ironfish-rust-nodejs/src/multisig.rs

```diff
@@ -415,6 +432,7 @@ pub fn dkg_round2(
         &round1_public_packages,
         thread_rng(),
     )
+    .map_err(|e| IronfishError::new_with_source(IronfishErrorKind::FrostLibError, e))
```

Comment:
I think we should look into adding fmt::Display for the errors returned from the frost lib in std mode, then I think we can skip spamming these map_err everywhere. Doesn't need to happen right now, just thinking out loud.

Reply:
I think we do have fmt::Display in std mode 🤔 

I'll see if we can easily get rid of these